.github/actions/rust/action.yaml

```diff
@@ -5,66 +5,68 @@ inputs:
   toolchain:
     description: Rust toolchain to use, stable / nightly / beta, or exact version
     # The same as in /README.md
-    default: "1.73"
+    default: "stable"
   target:
     description: Target Rust platform
     required: false
-
+    default: x86_64-unknown-linux-gnu
   components:
     description: List of additional Rust toolchain components to install
     required: false
+  cache:
+    description: Enable Rust cache
+    required: false
+    default: 'true'
+
 runs:
   using: composite
   steps:
+    # TODO: Move to AMI and build every day
     - uses: dtolnay/rust-toolchain@master
       name: Install Rust toolchain
       with:
         toolchain: ${{ inputs.toolchain }}
         target: ${{ inputs.target }}
         components: ${{ inputs.components }}
 
+    # TODO: Move to AMI and build every day
     - name: Install protoc
       id: deps-protoc
       shell: bash
       run: |
         curl -Lo /tmp/protoc.zip \
-          https://github.com/protocolbuffers/protobuf/releases/download/v22.0/protoc-22.0-linux-x86_64.zip
+          https://github.com/protocolbuffers/protobuf/releases/download/v22.0/protoc-22.0-linux-aarch_64.zip
```

Comment:
Will it work on different architectures? maybe we should add some if depending on runner arch?

Reply:
It works this way according to the tests. We want to move this step to AMI. When infra guys have time they will do it.